srcpkgs/libchewing/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'libchewing'
+pkgname=libchewing
+version=0.5.1
+revision=1
+#wrksrc=
+#create_wrksrc=yes
+#only_for_archs="i686 x86_64"
+build_style=cmake
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends="autoconf automake libtool clang gcc"
+makedepends="sqlite-devel"
+depends="sqlite"
+short_desc="An intelligent phonetic (Zhuyin/Bopomofo) input method"
+maintainer="Robert Stancil <robert.stancil@mavs.uta.edu>"
+license="BSD-3-Clause, LGPL-2.1-only"
+homepage="http://chewing.im/"
+distfiles="https://github.com/chewing/libchewing/releases/download/v${version}/${pkgname}-${version}.tar.bz2"
+checksum="9708c63415fa6034435c0f38100e7d30d0e1bac927f67bec6dfeb3fef016172b"
+shlib_provides="libchewing.so.3"
```

Comment:
I don't think this is necessary.

Reply:
I think this is an artifact of misunderstanding the comon/shlibs file. Removed

Reply:
This is only necessary when you need to provide shlib tracking for internal libraries (those that live under /usr/lib/pkgname/lib.so instead of /usr/lib/lib.so) See `mutter` as an example